tests/ragger/test_eip712.py

```diff
@@ -47,10 +47,10 @@ def test_eip712_legacy(app_client: EthereumClient):
     assert r == bytes.fromhex("ea66f747173762715751c889fea8722acac3fc35db2c226d37a2e58815398f64")
     assert s == bytes.fromhex("52d8ba9153de9255da220ffd36762c0b027701a3b5110f0a765f94b16a9dfb55")
 
-
 def test_eip712_new(app_client: EthereumClient, input_file: Path, verbose: bool, filtering: bool):
-    print("=====> %s" % (input_file))
-    if app_client._client.firmware.device != "nanos":
+    if app_client._client.firmware.device == "nanos":
+        pytest.skip("Not supported on LNS")
+    else:
```

Comment:
We have a fixture to tag test for specific backends, but as I'm not sure it's stable we can skip and keep in mind